helper/cli.c

```diff
@@ -221,8 +221,15 @@ static int cli_log_va(odp_log_level_t level, const char *fmt, va_list in_args)
 	 * string after the last newline and use it the next time we're called.
 	 */
 	va_copy(args, in_args);
-	len = vsnprintf(NULL, 0, fmt, args) + 1;
+	len = vsnprintf(NULL, 0, fmt, args);
 	va_end(args);
+
+	if (len < 0) {
+		ODPH_ERR("vsnprintf failed\n");
+		goto out;
+	}
+
+	len++;
 	str = malloc(len);
```

Comment:
`str = malloc(++len);`?

Reply:
That would be equivalent, yes. But I want it to be on display on its own line, it's too important to hide inside parenthesis in the malloc call.